core/napi.rs

```diff
@@ -0,0 +1,502 @@
+#![allow(non_camel_case_types)]
+#![allow(non_upper_case_globals)]
+
+pub use std::ffi::CStr;
+pub use std::mem::transmute;
+pub use std::os::raw::c_char;
+pub use std::os::raw::c_void;
+pub use std::ptr;
```

Comment:
Why are these reexported if they come from `std`?

Reply:
Convenience, these are used a lot in `cli/napi` made available with `use deno_core::napi::*`